eventrouter.c

```diff
@@ -712,8 +486,7 @@ void ErReturnToSender(ErEvent_t *a_event)
         // All the recipient tasks are done with the event. We must return the
         // event to its sender.
 
-        const size_t sending_task_index =
-            GetIndexOfTaskThatOwnsModule(a_event->m_sending_module);
+        const size_t sending_task_index = a_event->m_sending_module->m_task_idx;
```

Comment:
Question: what is motivation of the difference between `index` and `idx`?

Reply:
...I have no clue. I tend to use `*idx` for loop variables but I have no clue why I spelled out "index" in most other places. Updating index -> idx.